pkg/apis/core/v1beta1/helper/helper.go

```diff
@@ -92,13 +76,16 @@ func UpdatedCondition(condition gardencorev1beta1.Condition, status gardencorev1
 		Reason:             reason,
 		Message:            message,
 		LastTransitionTime: condition.LastTransitionTime,
-		LastUpdateTime:     Now(),
+		LastUpdateTime:     condition.LastUpdateTime,
 		Codes:              codes,
 	}
 
 	if condition.Status != status {
-		newCondition.LastTransitionTime = Now()
+		now := Now()
+		newCondition.LastTransitionTime = now
+		newCondition.LastUpdateTime = now
```

Comment:
The LastUpdate and LastTransitionTime should be updated only under specific conditions e.g. update to status / reason or message:

https://github.com/gardener/gardener/blob/24334336a6f069b00ad5c7b839fac980aa9ea5f8/pkg/apis/core/v1beta1/helper/condition_builder.go#L100-L155

Reply:
done